src/solidity/decodeInfo.js

```diff
@@ -254,12 +254,12 @@ function getStructMembers (type, stateDefinitions, contractName, location) {
   * @return {String} returns the token type (used to instanciate the right decoder) (uint[2] storage ref[2] will return 'array', uint256 will return uintX)
   */
 function typeClass (fullType) {
-  if (fullType.indexOf(']') !== -1) {
-    return 'array'
-  }
   if (fullType.indexOf('mapping') === 0) {
```

Comment:
I guess we have to distinguish `mapping(uint=>uint)[]` and `mapping(uint=>uint[])`, which this code currently does not.